package.json

```diff
@@ -37,7 +37,6 @@
   "peerDependencies": {
     "@stoplight/markdown-viewer": "^4",
     "@stoplight/ui-kit": "^3",
```

Comment:
Do we still rely on UI-Kit in JSV?

Reply:
Unfortunately it's still a peer of MDV :\

Reply:
I removed the include from the storybook `_styles.scss`. That at least make Storybook rebuilds faster.... which is a win I guess... 😅